ext/net/02_tls.js

```diff
@@ -131,6 +137,36 @@ class TlsListener extends Listener {
   }
 }
 
+function hasTlsKeyPairOptions(options) {
+  return (ReflectHas(options, "cert") || ReflectHas(options, "key") ||
```

Comment:
This change made a regression. Previously, if `cert` and `key` in options had `undefined` value, it server was not failing but now it does fail.

Context: https://github.com/unjs/nitro/pull/2373 -- tools might infer these options from sources like environment variables. now need to explicitly call `delete` on prop.